Assets/MixedRealityToolkit-SDK/Profiles/DefaultMixedRealitySpatialAwarenessProfile.asset

```diff
@@ -26,7 +28,7 @@ MonoBehaviour:
   meshOcclusionMaterial: {fileID: 2100000, guid: a809c7ea182b699409443d46d5b42435,
     type: 2}
   useSurfaceFindingSystem: 0
-  surfaceFindingPhysicsLayer: 31
+  surfaceFindingPhysicsLayer: 30
```

Comment:
Not fond of having a separate layer defined for surfaces vs meshes as there are only 24 user layers. That said, we should get customer feedback on the usability rather than block on this.

Reply:
We can make them the same layer. We didn't prevent them from being the same, but if users want to make them different they can.

Reply:
The nice thing about this too is that if the layer is already set then it moves it to the next availible layer.

Reply:
I'm not sure if you noticed that.